### DISCUSSION TITLE
A GUI when playing content on Tribler

### DISCUSSION THREAD

**Discussion by CrackersTie36** [Tribler/tribler#6146] [Ideas]
If a user wants to play a film or audio file that they downloaded through Tribler pls have a UI.

**Comment by ichorid** [comment]
You mean, like a set of buttons to play a video or a song?

**Reply by CrackersTie36** [reply]
Maybe sure, but my idea was like: if you double click an audio file or a film torrent it should open its own window playing the film/audio file. So like other torrenting clients you can play a film or an audio file within that app maybe you should add something like that into Tribler.

**Reply by ichorid** [reply]
Which other clients does it? Could you please provide a screenshot or just point to some examples?

**Reply by CrackersTie36** [reply]
![Screenshot 2021-06-01 210526](https://user-images.githubusercontent.com/48649220/120313609-c25caa80-c2c9-11eb-84a9-ea2822e8ace9.png)

**Reply by ichorid** [reply]
Do you play while you're still downloading? What is the typical GB size of a movie you download?

**Reply by CrackersTie36** [reply]
No, not normally. Typical GB size for a movies I normally download is around 1-2GBs
